src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,7 +1,40 @@
 package core.basesyntax;
 
+import java.io.IOException;
+import java.nio.charset.StandardCharsets;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.util.stream.Stream;
+
 public class WorkWithFile {
+
     public void getStatistic(String fromFileName, String toFileName) {
+        Path inputFile = Paths.get(fromFileName);
+        final int[] supply = {0};
+        final int[] buy = {0};
```

Comment:
Using arrays for holding a single value is not recommended when the value is not meant to be shared across threads or for the sake of finality. Consider using local variables instead.